javav2/example_code/support/Readme.md

```diff
@@ -0,0 +1,70 @@
+- [Get and display Support severity levels](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/support/src/main/java/com/example/support/SupportScenario.java) (describeSeverityLevels command)
+- [Get open support cases](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/support/src/main/java/com/example/support/SupportScenario.java) (describeCases command)
+- [List the communications of the support case](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/support/src/main/java/com/example/support/SupportScenario.java) (describeCommunications command)
+- [Resolve the support case](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/support/src/main/java/com/example/support/SupportScenario.java) (resolveCase command)
+
+### Scenario 
+
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- [Perform AWS Support operations](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/support/src/main/java/com/example/support/SupportScenario.java) (Multiple commands)
+
+## Run the AWS Support Java file
+
+**Be very careful** when running an operation that deletes or modifies AWS resources in your account. We recommend creating separate test-only resources when experimenting with these examples.
+
+To run these examples, set up your development environment. For more information, 
+see [Get started with the SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup.html). 
+
+
+ ## Test the AWS Support Java file
+
+You can test the Java code example for AWS Support by running a test file named **SupportTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
+
+You can run the JUnit tests from an IDE, such as IntelliJ, or from the command line. As each test runs, you can view messages that inform you if the various tests succeed or fail. For example, the following message informs you that Test 3 passed.
+
+	Test 3 passed
+
+**WARNING**: _Running these JUnit tests manipulates real AWS resources and might incur charges on your account._
+
+ ### Properties file
+Before running the JUnit tests, you must define values in the **config.proerties** file located in the **resources** folder. This file contains values that are required to run the JUnit tests. If you do not define all values, the JUnit tests fail.
+
+Define this value to successfully run the JUnit tests:
+
+- **fileAttachment** - The file can be a simple saved .txt file to use as an email attachment.  
+
+## Additional resources
+* [Developer Guide - AWS SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html).
+* [Developer Guide - AWS Support](https://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
```

Comment:
Also add a link to AWS Support reference for the Java SDK, maybe Support package? Whatever makes the most sense:
https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/support/package-summary.html

Reply:
fixed